### PR TITLE
Compatibility for png export in bokeh 0.12.7

### PR DESCRIPTION
**Pull request by philippjfr**
Minor fix for API change for png export in bokeh 0.12.7devX.

### CODE REVIEW

**Comment by jlstevens** [comment]
Looks good. You could maybe set up a unit test to check for any output when requesting a PNG from Bokeh. But then we would probably need to update Travis with the new dependencies.

I'll leave that up to you but I am happy to merge.

**Comment by philippjfr** [comment]
>But then we would probably need to update Travis with the new dependencies.

Right that would probably require rebuilding the tests.

**Comment by jlstevens** [comment]
Right. Might be worth doing later and if you agree just file an issue as a reminder. Merging.

**Comment by philippjfr** [comment]
>In that case it might be worth doing later and if you agree just file an issue as a reminder. Merging.

At a later point we could even consider having some bokeh display tests which use png export.

**Comment by jlstevens** [comment]
> At a later point we could even consider having some bokeh display tests which use png export.

Agreed. Probably worth filing an issue. Not sure if the output is byte-by-byte consistent but we would certainly find out!

File: holoviews/plotting/bokeh/renderer.py [hunk 1]
```diff
@@ -82,7 +82,10 @@ def __call__(self, obj, fmt=None, doc=None):
             if bokeh_version < '0.12.6':
                 raise RuntimeError('Bokeh png export only supported by versions >=0.12.6.')
             from bokeh.io import _get_screenshot_as_png
-            img = _get_screenshot_as_png(plot.state)
+            if bokeh_version > '0.12.6':
+                img = _get_screenshot_as_png(plot.state, None)
+            else:
+                img = _get_screenshot_as_png(plot.state)
```
Comment (bryevdv) [inline holoviews/plotting/bokeh/renderer.py]: Well this sucks, I wish you'd mentioned you were using a private API, perhaps we could have made a public one with better guarantees

Comment (philippjfr) [inline holoviews/plotting/bokeh/renderer.py]: Yes, should have mentioned it, it's my fault. I was going to bring it up but we needed something that would work for this dev release and I only noticed last night.

**Pull request merged**